aspects/schema.go

```diff
@@ -845,18 +845,26 @@ func (v *arraySchema) parseConstraints(constraints map[string]json.RawMessage) e
 
 func (v *arraySchema) expectsConstraints() bool { return true }
 
-type validationError struct {
-	path    []string
+type ValidationError struct {
+	Path    []string
 	baseErr error
```

Comment:
maybe this should also be export, then following the stdlib, it should just be called Err